students/11th/leeyongmin/js/main.js

```diff
@@ -0,0 +1,134 @@
+// post comment by click
+const postComment = document.getElementsByClassName("first-feeds-postComment");
+const commentButton = postComment[0].children[1];
+const commentTextarea = postComment[0].children[0];
+
+commentButton.addEventListener("click", () => {
+  const commentList = document.getElementsByClassName("first-feeds-comments");
+
+  const commentValue = commentTextarea.value;
+
+  if (commentValue === "") {
+    alert("no comment");
+    return;
+  }
+
+  const listItem = document.createElement("li");
+  const likeImg = document.createElement("img");
+
+  likeImg.setAttribute(
+    "src",
+    "https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/heart.png"
+  );
+  listItem.innerHTML = `<span>위코드</span>${commentValue}`;
+  listItem.appendChild(likeImg);
+  commentList[0].appendChild(listItem);
+  commentTextarea.value = "";
+});
+
+// post comment by enter
+commentTextarea.addEventListener("keyup", (e) => {
+  if (e.key === "Enter") {
+    const commentList = document.getElementsByClassName("first-feeds-comments");
+    const listItem = document.createElement("li");
+    const likeImg = document.createElement("img");
+    const commentValue = commentTextarea.value;
+    if (commentValue.length <= 1) {
+      alert("no comment");
+      return;
+    }
+    likeImg.setAttribute(
+      "src",
+      "https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/heart.png"
+    );
+    listItem.innerHTML = `<span>위코드</span>${commentValue}`;
+    listItem.appendChild(likeImg);
+    commentList[0].appendChild(listItem);
+    commentTextarea.value = null;
+  }
+});
+
+// 스토리 모두보기
+const storyButton = document.getElementsByClassName(
+  "main-right-story-header"
+)[0].children[1];
+
+storyButton.addEventListener("click", () => {
+  const storyWindow = document.getElementsByClassName("main-right-story")[0];
+  if (storyButton.innerHTML === "모두 보기") {
+    storyWindow.style.height = "auto";
+    storyButton.innerHTML = "접기";
+  } else {
+    storyWindow.style.height = "300px";
+    storyButton.innerHTML = "모두 보기";
+  }
```

Comment:
```suggestion
  const showAll = storyButton.innerHTML === "모두 보기";
  storyWindow.style.height = showAll ? "auto" : "300px";
  storyButton.innerHTML = showAll ? "접기" : "모두 보기";
```

이런 방법도 있습니다. 참고해주세요. 항상 줄인다고 좋은 건 아니고 가독성 좋은 코드가 좋은 코드입니다.